languages/translations/es.json

```diff
@@ -54,7 +54,8 @@
             "two linked by distance": "{instruction_one} y luego en {distance}, {instruction_two}",
             "two linked": "{instruction_one} y luego {instruction_two}",
             "one in distance": "A {distance}, {instruction_one}",
-            "name and ref": "{name} ({ref})"
+            "name and ref": "{name} ({ref})",
+            "exit with number": "la salida {exit}"
```

Comment:
The article _la_ should be omitted from this string. It wouldn’t appear in signage.

[![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/bd/PR-66_este_hacia_PR-3.jpg/1024px-PR-66_este_hacia_PR-3.jpg)](https://commons.wikimedia.org/wiki/File:PR-66_este_hacia_PR-3.jpg)

Reply:
👍